rhobot_test.go

```diff
@@ -30,9 +30,9 @@ func TestLogLevelingFiltering(t *testing.T) {
 	}
 	originalReportSet := report.Set{Elements: elements, Metadata: metadata}
 	TestLogLevelResults := map[string]int{
-		"Debug": 5,
-		"Info":  4,
-		"Warn":  3,
+		"Debug": 6,
+		"Info":  5,
+		"Warn":  4,
```

Comment:
There was an extra "Warn" level healthcheck in "healthchecksAll.yml" that previously was never counted in the reports because it was preceded by a fatal healthcheck. Since the behaviour of reports has been changed to no longer omit elements after a fatal, it is now included in the "Debug", "Info", and "Warn" level reports and the map "TestLogLevelResults" now reflects that.